x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_settings_management/index.tsx

```diff
@@ -0,0 +1,262 @@
+      return;
+    }
+    closeConfirmModal();
+    handleSave(true);
+    setConversationsSettingsBulkActions({});
+  }, [
+    closeConfirmModal,
+    conversationsSettingsBulkActions,
+    handleSave,
+    setConversationsSettingsBulkActions,
+  ]);
+
+  const onDeleteCancelled = useCallback(() => {
+    setDeletedConversation(null);
+    closeConfirmModal();
+    onCancelClick();
+  }, [closeConfirmModal, onCancelClick]);
+
+  const { getConversationsList, getColumns } = useConversationsTable();
+
+  const conversationOptions = getConversationsList({
+    allSystemPrompts,
+    actionTypeRegistry,
+    connectors,
+    conversations: conversationSettings,
+    defaultConnector,
+  });
+
+  const onSaveCancelled = useCallback(() => {
+    closeEditFlyout();
+    onCancelClick();
+  }, [closeEditFlyout, onCancelClick]);
+
+  const onSaveConfirmed = useCallback(() => {
+    closeEditFlyout();
+    handleSave(true);
+    setConversationsSettingsBulkActions({});
+  }, [closeEditFlyout, handleSave, setConversationsSettingsBulkActions]);
+
+  const columns = useMemo(
+    () =>
+      getColumns({
+        conversations: conversationSettings,
+        onDeleteActionClicked,
+        onEditActionClicked,
+        allSystemPrompts,
+      }),
+    [allSystemPrompts, conversationSettings, getColumns, onDeleteActionClicked, onEditActionClicked]
+  );
+
+  const sorting = useMemo(
+    () => ({
+      sort: {
+        field: 'updatedAt',
+        direction: 'desc' as const,
+      },
+    }),
+    []
+  );
+
+  const confirmationTitle = useMemo(
+    () =>
+      deletedConversation?.title
+        ? i18n.DELETE_CONVERSATION_CONFIRMATION_TITLE(deletedConversation?.title)
+        : i18n.DELETE_CONVERSATION_CONFIRMATION_DEFAULT_TITLE,
+    [deletedConversation?.title]
+  );
+
+  const pagination = useMemo(
+    () => ({
+      pageIndex: DEFAULT_PAGE_INDEX,
+      pageSize: DEFAULT_PAGE_SIZE,
+      pageSizeOptions: [DEFAULT_PAGE_SIZE],
+      totalItemCount: conversationOptions.length,
+    }),
+    [conversationOptions.length]
+  );
+
+  if (!conversationsLoaded) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiPanel hasShadow={false} hasBorder paddingSize="l">
+        <ConversationStreamingSwitch
+          assistantStreamingEnabled={assistantStreamingEnabled}
+          setAssistantStreamingEnabled={setAssistantStreamingEnabled}
+          compressed={false}
+        />
+        <EuiSpacer size="m" />
+        <EuiInMemoryTable
+          items={conversationOptions}
+          columns={columns}
+          pagination={pagination}
+          sorting={sorting}
+        />
+      </EuiPanel>
+      {editFlyoutVisible && (
+        <Flyout
+          flyoutVisible={editFlyoutVisible}
+          onClose={onSaveCancelled}
+          onSaveConfirmed={onSaveConfirmed}
+          onSaveCancelled={onSaveCancelled}
+          title={selectedConversation?.title ?? i18n.CONVERSATIONS_TABLE_COLUMN_CONVERSATIONS}
+        >
```

Comment:
Can we set a max-width on the flyout? Or use a [fixed-width variant](https://eui.elastic.co/#/layout/flyout%23sizing)? I don't think it needs to stretch across the whole page if there's that much width? cc @bojanasan 

<img width="2508" alt="image" src="https://github.com/elastic/kibana/assets/2946766/6bea9d59-bf7c-41b6-81fb-107e8dcf7fa8">

Reply:
Max width now set to 656px:

<img width="2558" alt="Screenshot 2024-06-28 at 17 23 55" src="https://github.com/elastic/kibana/assets/6295984/a1b407a7-e134-4076-8b4c-ebb2587b4d00">